src/Microsoft.NET.Sdk.Functions.Generator/MethodInfoExtensions.cs

```diff
@@ -77,6 +83,41 @@ public static FunctionJsonSchema ToFunctionJson(this MethodDefinition method, st
             };
         }
 
+        /// <summary>
+        /// Gets bindings from return expression used with a binding expression.
+        /// Ex:
+        ///     [FunctionName("HttpTriggerWriteToQueue1")]
+        ///     [return: Queue("myqueue-items-a", Connection = "MyStorageConnStra")]
+        ///     public static string Run([HttpTrigger] HttpRequestMessage request) => "foo";
+        /// </summary>
+        private static JObject[] GetOutputBindingsFromReturnAttribute(MethodDefinition method)
```

Comment:
I was going to say that a return should/can only be bound once, but it looks like in this generator, we're handling 1->N (e.g. [here](https://github.com/Azure/azure-functions-vs-build-sdk/blob/b0e54a832a92119e00a2b1796258fcf88e0d6109/src/Microsoft.NET.Sdk.Functions.Generator/ParameterInfoExtensions.cs#L23)). I'd expect at runtime that this would fail however, since parameter/return bindings must be 1:1. Not sure if we actually have explicit validation for this in the host or SDK.